script/script.js

```diff
@@ -22,3 +22,34 @@ if (localStorage.getItem('formData')) {
   titleInput.value = formValueObj.title;
   authorInput.value = formValueObj.author;
 }
+
+const bookLink = document.getElementById('library');
+const bookStore = document.getElementById('bookStore');
+const addBook = document.getElementById('addBook');
```

Comment:
Please follow [this](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) wireframe for the books list as it's mentioned in the requirements.

Reply:
Hello @SajjadAhmad14, I haven't understand specifically what changes I should make. Can you please explain it thoroughly if you don't mind?

Reply:
Hi @AbdimulhinYussuf3675, I think what the previous reviewer is saying is that, as much as you tried personalizing your design, your design should still stick to the wireframe given. But I can see your list of books is looking completely different from the wireframe given. You have done a great job implementing all other features though and I liked how you personalized the colors. Keep up the good work.